app/controllers/api/v1/games_controller.rb

```diff
@@ -1,6 +1,8 @@
 module Api
   module V1
     class GamesController < ActionController::API
+      rescue_from ActiveRecord::RecordNotFound, with: :record_not_found
```

Comment:
@icorson3, @jmejia, I tried this method in both API controller and application controller and neither worked. File structure looks like this
<img width="281" alt="screen shot 2018-05-17 at 7 09 40 pm" src="https://user-images.githubusercontent.com/5290130/40211116-fcc860b8-5a05-11e8-8079-b96ceb28e0b4.png"> ... any guidance on what's wrong?

Reply:
@ssciolist I've never used `rescue_from`.  What's your goal here? Also not sure what it means for it to not work. Can you clarify?